api/python/provisioner/commands/set_swupdate_repo.py

```diff
@@ -249,6 +249,9 @@ def dynamic_validation(self, params, targets: str, dry_run: bool = False):  # no
                     str(repo.source), f"malformed repo: '{exc}'"
                 )
 
+            # FIXME should respect different version of iso,
```

Comment:
This comment is not clear since the `set_swupgrade_repo` command has support of two existing ISO versions.

Reply:
yes, but that comment was for older logic, I removed that comment since it was wrong actually: legacy `sw_update` logic shouldn't support that